src/Core/Containers/AdjacencyList.hpp

```diff
@@ -45,8 +45,10 @@ class RA_CORE_API AdjacencyList
     //////////////////////////////////////////////////////////////////////////////
     // NODE
     //////////////////////////////////////////////////////////////////////////////
-    /// Return the index of the added leaf. Use -1 to create the root node.
-    uint addNode( const int parent );
+    /// Return the index of the added root.
```

Comment:
Can multiple roots be added ?
What happens when this method is called multiple times ?

Reply:
one can add several roots, each will just have no parent and be at graph level 0.